spec/models/patient_spec.rb

```diff
@@ -1,7 +1,9 @@
 require "rails_helper"
 
 describe Patient, type: :model do
+  let(:reporting_timezone) { CountryConfig.current[:time_zone] }
```

Comment:
How is this config different from `ANALYTICS_TIME_ZONE`?

Reply:
Oof, great question. It's not/shouldn't be. We've got:
* `ANALYTICS_TIME_ZONE` env var
* `CountryConfig.current[:time_zone]` (from country configurations, based on `DEFAULT_COUNTRY` env var)
* ...and `Period::ANALYTICS_TIME_ZONE`, which is CountryConfig with a default value

I'm not sure if we should use CountryConfig here, or Period here, but I'm fairly certain we can start phasing out `ANALYTICS_TIME_ZONE`.

Reply:
Fixed in 1aa47d9254236c297e059e239b7e53dd6056ddab. We use `Period::ANALYTICS_TIME_ZONE` everywhere.